openvpn/middlewares/server/auth/middleware_test.go

```diff
@@ -91,122 +74,90 @@ func Test_ConsumeLineTakes(t *testing.T) {
 
 func Test_ConsumeLineAuthState(t *testing.T) {
 	var tests = []struct {
-		line          string
-		expectedState openvpn.State
+		line string
 	}{
-		{">CLIENT:REAUTH,0,0", openvpn.STATE_AUTH},
-		{">CLIENT:CONNECT,0,0", openvpn.STATE_AUTH},
+		{">CLIENT:REAUTH,0,0"},
+		{">CLIENT:CONNECT,0,0"},
 	}
 
 	for _, test := range tests {
 		fas := newFakeAuthenticatorStub()
 		middleware := NewMiddleware(fas.fakeAuthenticator)
-		connection := &fakeConnection{}
-		middleware.Start(connection)
+		mockWritter := &management.MockConnection{}
+		middleware.Start(mockWritter)
```

Comment:
should be `mockWriter`

Reply:
Actually it should be mockConnection. Fixed, thanks for pointing out